docker.sh

```diff
@@ -25,11 +26,19 @@ print_next_version() {
   echo "rustvmm/dev:v$(next_version)"
```

Comment:
could use `$IMAGE_NAME` here:
```suggestion
  echo "$IMAGE_NAME:v$(next_version)"
```

Reply:
Ah, it was like this even before when we had `$DOCKER_TAG`